pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -422,7 +491,7 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 		}
 
 		if currNodeState.LastFailedRevision == revisionToStart && currNodeState.LastFailedTime != nil && !currNodeState.LastFailedTime.IsZero() {
-			delay := backOffDuration(currNodeState.LastFailedCount)
+			delay := c.backOffDuration(currNodeState.LastFailedCount)
```

Comment:
when `staticPodStateFallback` won't we enter `currNodeState.TargetRevision > currNodeState.CurrentRevision` (a few lines earlier) and install a pod again !?

Reply:
not sure what you mean. We check that the lastFailedTime is set and we wait at least the given backoff time.

What is missing here is that we need two backoffs, or we have to make this much longer. This was mainly used for the installer before.

Reply:
We could add a nextAttempt timestamp into the API. When writing the lastFailTime we know which case it is and we can compute a matching backoff. Wdyt?